sphinx/ext/autosummary/__init__.py

```diff
@@ -161,7 +160,7 @@ def autosummary_table_visit_html(self, node):
             par = cast(nodes.paragraph, col1_entry[0])
             for j, subnode in enumerate(list(par)):
                 if isinstance(subnode, nodes.Text):
-                    new_text = text_type(subnode.astext())
+                    new_text = str(subnode.astext())
```

Comment:
`Node.astext()` always returns `str` instance. So this conversion is not needed. Let's remove it.